### DISCUSSION TITLE
Start wt from explorer to open the current folder

### DISCUSSION THREAD

**Discussion by yk0n9** [microsoft/terminal#17271] [Q&A] [answered]
After entering ```wt``` in the address bar under ```D:/``` in ```explorer```, the address of the opened terminal is still C:/User/~ instead of the current folder. : ```D:/```

**Comment by Yewzir** [comment, marked as answer]
@yk0n9 

You can rightclick in any folder, and choose **Open in Terminal** from the context menu. It will launch Windows Terminal, starting in that particular folder, guaranteed.

![image](https://github.com/user-attachments/assets/9829a226-3c81-4c99-9918-6bf435bcbb22)
![image](https://github.com/user-attachments/assets/88792b01-2330-4d04-b985-ac1297b98bb8)

Edit: Launching Windows Terminal from the address bar, in the current folder, failed for me as well, don't worry.
However, I found that if I checked having to **use the parent process directory** in the settings of the application profile, it worked.

![image](https://github.com/user-attachments/assets/d804deae-7e15-451b-8afc-9089c66a1121)

🎉 

Kind regards
Yewzir
